src/src.pro

```diff
@@ -855,6 +855,7 @@ else:ios {
             tasks/purchase/taskpurchase.h
 
     OBJECTIVE_HEADERS += \
+            features/featureioskillswitch.h \
```

Comment:
move to the "HEADERS" block. I don't think it changes too much, but it's C++ and not obj-C++